code/datums/components/heat_sensitive.dm

```diff
@@ -0,0 +1,23 @@
+/datum/component/heat_sensitive
```

Comment:
whats the reasoning for making it a component instead of a element?

Reply:
smoool brain, would elements work here? I need to store some details about how to handle the object after all.

Reply:
My reasoning is that a element would have reduced memory use and you can use /static/ variable for object types to ensure the min and max heat is using as little memory as possible. There are a lot of items that people will want to make heat_sensitive.

Reply:
that's true, I'll give it a look. Thanks for explaining

Reply:
Yeah this is needed, about 9k objects with the component on meta, I'll give elements a look

Reply:
Fuck we gotta just live with it, I can get the tracker close to functional, but the actual heat reaction needs to store state of who registered it. I can lower the footprint of the max/min heat needed tho

Reply:
mmmmmmm saves about 0.7MB, not sure if it's worth it

Reply:
Is it possible to make the tracker be a part of the objects that uses a otherwise empty variable to store the info so its only existing if this object uses it?

Reply:
That's so fucking hacky man. I'll think about it